lib/xcprofiler.rb

```diff
@@ -28,6 +28,7 @@ def execute(args)
         opts.on("--derived-data-path", String, "Root path of DerivedData") { |v| options.derived_data_path = v }
         opts.on("-t", "--truncate-at [TRUNCATE_AT]", Integer, "Truncate the method name with specified length") { |v| options.truncate_at = v }
         opts.on("--[no-]unique", "Reject duplicated location results or not") { |v| options.unique = v }
+        opts.on("--output-path [PATH]", String, "File path to output reporters' result") { |v| options.output_path = v }
```

Comment:
Should be -o and --output.
At least -path is needless.

Reply:
`-o` is already used. So I renamed to `--output`